helm/cluster/templates/clusterapi/controlplane/README.md

```diff
@@ -0,0 +1,168 @@
+# Cluster API resources for the control plane
```

Comment:
Semi-generated / semi-typed table with all API server config, with the ope we use being highlighted.

Just trying this out, it's not pretty, it should probably done in some better way, but the point is to try to have docs for components which show what config are we using, what can be configured, etc.